README.md

```diff
@@ -683,3 +684,84 @@ export FISSILE_DOCKER_AUTH=basic
 make registry
 curl -u ${FISSILE_DOCKER_USERNAME}:${FISSILE_DOCKER_PASSWORD} https://registry.cf-dev.io:5000/v2/
 ```
+
+## Using Persi NFS
+
+
+### Running a test NFS server
+
+```bash
+# Enable NFS modules
+sudo modprobe nfs
+sudo modprobe nfsd
+
+docker run -d --name nfs \
+    -v "/home/toaster/tools/nfs_share:/exports/foo" \
+    -p 111:111/tcp \
+    -p 111:111/udp \
+    -p 662:662/udp \
+    -p 662:662/tcp \
+    -p 875:875/udp \
+    -p 875:875/tcp \
+    -p 2049:2049/udp \
+    -p 2049:2049/tcp \
+    -p 32769:32769/udp \
+    -p 32803:32803/tcp \
+    -p 892:892/udp \
+    -p 892:892/tcp \
+    --privileged \
+    viovanov/nfs-server /exports/foo
```

Comment:
We need sources for that :D

Reply:
I don't think we do - this is just a how-to on setting up a quick NFS server - we're not delivering any of it. But here it is: https://github.com/viovanov/nfs-server